noxfile.py

```diff
@@ -358,8 +358,7 @@ def build_dists(session: nox.Session) -> List[str]:
         )
 
     session.log("# Build distributions")
-    session.install("setuptools", "wheel")
-    session.run("python", "setup.py", "sdist", "bdist_wheel", silent=True)
+    session.run("python", "-m", "build", silent=True)
```

Comment:
Let's make this non-silent?

Reply:
I don't have a preference. Note it's really verbose, though (all the setuptools log, basically)

Reply:
Ah, nvm then!